src/components/custom-field-input-currency/custom-field-input-currency.jsx

```diff
@@ -73,6 +74,12 @@ export default function CustomFieldInputCurrency(props) {
     }
   });
 
+  useImperativeHandle(ref, () => ({
+    value: () => {
+      return [parseInt(componentRef.current.value().replace(/\D/g, ''), 10), props.currencyCode];
```

Comment:
I love that this works because virtually all currencies are in decimal!

Reply:
Even when they're not a `.` separator, they're `,`, which is covered by `\D`! 😄